spec/operators/bufferWhen-spec.ts

```diff
@@ -198,7 +198,7 @@ describe('bufferWhen operator', () => {
       const result = e1.pipe(
         bufferWhen(() => {
           if (i === 1) {
-            throw 'error';
+            return throwError(() => 'error');
```

Comment:
This complete changes the test. Revert.

Reply:
@benlesh Yeah I assumed this wasn't the right way to go about it, but did it on the off-chance that the test itself was previously incorrect (as I figured that `bufferWhen`'s `closingSelector` was supposed to return an observable).